src/hooks/useAxisWheelZoom.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  usePlotControls,
+  usePlotEvents,
+} from '../contexts/plotController/plotControllerContext';
+
+import { ControllerHookOptions } from './types';
+
+export interface UseAxisWheelZoomOptions extends ControllerHookOptions {
+  direction?: 'horizontal' | 'vertical';
+  axisId?: string;
+  zoomCenter?: 'mouse' | number;
+  zoomSpeed?: number;
+}
```

Comment:
We should add one more option `invert` by default 'false'. This allows to invert the direction of the scroll. The default behaviour should be opposite to the current one. Meaning when you scroll up the spectrum increase of size.

Reply:
prop added